app/repositories/SessionCacheRepository.scala

```diff
@@ -31,63 +31,66 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 import play.api.libs.json._
 import java.time.{Instant, ZoneOffset}
-import org.mongodb.scala.{SingleObservableFuture,ToSingleObservablePublisher}
+import org.mongodb.scala.{SingleObservableFuture, ToSingleObservablePublisher}
 
 @Singleton
-class DefaultSessionCacheRepository @Inject()(mongoComponent: MongoComponent,
-                                              config: Configuration)
-                                             (implicit executionContext: ExecutionContext)
-  extends PlayMongoRepository[AccountLinksMongo](
-    collectionName = "customs-financials-session-cache",
-    mongoComponent = mongoComponent,
-    domainFormat = AccountLinksMongo.format,
-    indexes = Seq(
-      IndexModel(
-        ascending("lastUpdated"),
-        IndexOptions().name("customs-financials-cache-last-updated-index")
-          .unique(true)
-          .expireAfter(config.get[Long]("mongodb.timeToLiveInSeconds"), TimeUnit.SECONDS)
+class DefaultSessionCacheRepository @Inject() (mongoComponent: MongoComponent, config: Configuration)(implicit
+  executionContext: ExecutionContext
+) extends PlayMongoRepository[AccountLinksMongo](
+      collectionName = "customs-financials-session-cache",
+      mongoComponent = mongoComponent,
+      domainFormat = AccountLinksMongo.format,
+      indexes = Seq(
+        IndexModel(
+          ascending("lastUpdated"),
+          IndexOptions()
+            .name("customs-financials-cache-last-updated-index")
+            .unique(true)
+            .expireAfter(config.get[Long]("mongodb.timeToLiveInSeconds"), TimeUnit.SECONDS)
+        )
       )
-    )) with SessionCacheRepository {
+    )
+    with SessionCacheRepository {
 
-  override def get(sessionId: String, linkId: String): Future[Option[AccountLink]] = {
+  override def get(sessionId: String, linkId: String): Future[Option[AccountLink]] =
     for {
       record <- collection.find(equal("_id", sessionId)).toSingle().toFutureOption()
-      result = record.flatMap(_.accountLinks.find(_.linkId == linkId))
+      result  = record.flatMap(_.accountLinks.find(_.linkId == linkId))
     } yield result
-  }
 
-  override def verifySessionId(sessionId: String): Future[Boolean] = {
+  override def verifySessionId(sessionId: String): Future[Boolean] =
     for {
       record <- collection.find(equal("_id", sessionId)).toSingle().toFutureOption()
     } yield record.nonEmpty
-  }
 
   override def getAccountLinks(sessionId: String): Future[Option[Seq[AccountLink]]] = {
     val target = 7
 
     for {
       record <- collection.find(equal("_id", sessionId)).toSingle().toFutureOption()
-      result = record.flatMap(a => Option(a.accountLinks.filter(link => link.accountNumber.length == target)))
+      result  = record.flatMap(a => Option(a.accountLinks.filter(link => link.accountNumber.length == target)))
```

Comment:
You have added a double space here before the equals, is that intentional?

Reply:
That's part of scalafmt align settings